static/_redirects

```diff
@@ -209,19 +207,20 @@
 /docs/reference/glossary/maintainer/    /docs/reference/glossary/approver/ 301
 
 /docs/reference/kubectl/kubectl/kubectl_*.md    /docs/reference/generated/kubectl/kubectl-commands#:splat 301
-/docs/reference/workloads-18-19/     https://v1-9.docs.kubernetes.io/docs/reference/workloads-18-19/ 301
+# /docs/reference/workloads-18-19/     https://v1-9.docs.kubernetes.io/docs/reference/workloads-18-19/ 301
 
 /docs/reporting-security-issues/     /security/ 301
 
-/docs/resources-reference/1_6/*     /docs/resources-reference/v1.6/ 301
-/docs/resources-reference/1_7/*     /docs/resources-reference/v1.7/ 301
-/docs/resources-reference/v1.8/*     /docs/api-reference/v1.8/:splat 301
+# /docs/resources-reference/1_6/*     /docs/resources-reference/v1.6/ 301
+# /docs/resources-reference/1_7/*     /docs/resources-reference/v1.7/ 301
+# /docs/resources-reference/v1.8/*     /docs/api-reference/v1.8/:splat 301
```

Comment:
If you're happy to remove these lines entirely, feel free.

Reply:
🆗